src/components/Channel/ChatForm.jsx

```diff
@@ -0,0 +1,83 @@
+import { useState, useEffect } from 'react';
+import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
+
+import { socketClient } from '../../hooks/socket/useSocket';
+
+export default function ChatForm({ channelId }) {
+  const [input, setInput] = useState('');
+  const [userName, setUserName] = useState('');
+
+  useEffect(() => {
+    const { name } = JSON.parse(sessionStorage.getItem('user'));
+    setUserName(name);
+  }, []);
+
+  function submitChat(ev) {
+    ev.preventDefault();
+
+    const trimmedInput = input.trim();
+
+    if (!trimmedInput) return;
+
+    const newChat = {
+      author: userName.replace(' ', ''),
+      chat: trimmedInput,
+    };
+
+    socketClient.emit('new chat', { channelId, newChat });
+    setInput('');
+  }
+
+  function handleInputChange({ target }) {
+    const { value } = target;
+    setInput(value);
+  }
+  return (
+    <Form>
+      <input
+        type="text"
+        value={input}
+        autoComplete="off"
+        onChange={handleInputChange}
```

Comment:
onChange시에 하는 일이 단순히 input의 state를 업데이트하는 일이라면 
```suggestion
        onChange={(e) => setInput(e.target.value)}
```
이렇게만 작성해도 될 것 같습니다.